WordPress/UITestsFoundation/XCUIElement+Scroll.swift

```diff
@@ -21,18 +21,4 @@ extension XCUIElement {
             XCTFail("Unable to scroll element into view")
         }
     }
-
-    // Taken from https://stackoverflow.com/a/46943935
-    /// Scroll an element to the screen top.
-    func scrollToTop() {
-        let topCoordinate = XCUIApplication().screenTopCoordinate
-        let elementCoordinate = coordinate(withNormalizedOffset: .zero)
-
-        // Adjust coordinate so that the drag is straight up, otherwise
-        // an embedded horizontal scrolling element will get scrolled instead
-        let delta = topCoordinate.screenPoint.x - elementCoordinate.screenPoint.x
-        let deltaVector = CGVector(dx: delta, dy: 0.0)
-
-        elementCoordinate.withOffset(deltaVector).press(forDuration: 0.1, thenDragTo: topCoordinate)
-    }
```

Comment:
This is no longer used due to [this](https://github.com/wordpress-mobile/WordPress-iOS/pull/20796/files#r1214378290).